priv/templates/nimble_template/.github/workflows/test.yml.eex

```diff
@@ -87,22 +144,254 @@ jobs:
 
       - name: Run codebase check
         run: mix codebase
+    
+  seed_data:
+    name: Test seed data
+    
+    needs: lint_codebase
+    
+    runs-on: ubuntu-latest
+
+    services:
+      db:
+        image: postgres:12.3
+        ports: ['5432:5432']
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    
+    env:
+      MIX_ENV: dev
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Set up Elixir
+        uses: erlef/setup-elixir@v1
+        with:
+          otp-version: ${{ env.OTP_VERSION }}
+          elixir-version: ${{ env.ELIXIR_VERSION }}
+      <%= if web_project? do %>
+      - name: Set up Node
+        uses: actions/setup-node@v2.1.5
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      <% end %>
+      - name: Cache Elixir build
+        uses: actions/cache@v2
+        with:
+          path: |
+            _build
+            deps
+          key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-mix-
+      <%= if web_project? do %>
+      - name: Cache Node modules
+        uses: actions/cache@v2
+        with:
+          path: assets/node_modules
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      <% end %>
+      - name: Install Elixir dependencies
+        run: mix deps.get
+
+      - name: Compile dependencies
+        run: mix compile
+
+      <%= if web_project? do %>
+      - name: Install Node dependencies
+        run: npm --prefix assets install
+
+      - name: Compile assets
+        run: npm run --prefix assets build:dev
+      <% end %>
+      - name: Create database
+        run: mix ecto.create
+
+      - name: Migrate database
+        run: mix ecto.migrate
+
+      - name: Seed data
```

Comment:
How about:

```suggestion
      - name: Seed database
```

Reply:
Updated in https://github.com/nimblehq/elixir-templates/pull/163/commits/8c6197c1f715061cc3261c1ac2c284a90c71e079